app/controllers/concerns/resource_controller.rb

```diff
@@ -59,21 +59,9 @@ def edit
   end
 
   def update
-    @change_set = change_set_class.new(find_resource(params[:id])).prepopulate!
+    @change_set = new_change_set.prepopulate!
     authorize! :update, @change_set.resource
-    if @change_set.validate(resource_params)
```

Comment:
I think these extractions make this method harder to parse. What are other's thoughts?

Reply:
I find this diff very hard to read, but looking at the end state, the `update` method looks very clean and simple to me.

I'm not sure extracting the `change_set` and `new_change_set` methods is worth it (in fact, I couldn't find any place that's using the new `change_set` method which line 62 duplicates).

Reply:
Yeah I guess it feels more straightforward to have the logic in the update action if it fits. Did rubocop complain?

Reply:
Not about this restructuring, but I can return to the original form of the `#update` action.